content/course-specific-material/introR-SAWSC-Feb17.Rmd

```diff
@@ -67,4 +67,41 @@ Andrew Yan (<ayan@usgs.gov>)
 
 ### Lesson scripts
 
-Instructors will be live coding during this course. Our code will be shared with you here at the end.
\ No newline at end of file
+Instructors will be live coding during this course. Our code will be shared with you here at the end.
+
+### smwrQW 
+
+`smwrQW` is a package used for analyzing censored water quality data. It does not currently have a named maintainer, but there will be community efforts to address questions and fix bugs on [smwrQW's GitHub page](https://github.com/USGS-R/smwrQW/issues) in a timely fashion. The package will remain available for use on GitHub and GRAN.
```

Comment:
I'd change to " but there are community efforts to address questions and fix bugs"